src/steps/run_state.go

```diff
@@ -74,6 +106,15 @@ func (runState *RunState) CreateUndoRunState() (result RunState) {
 	return
 }
 
+// Save saves the run state to disk
+func (runState *RunState) Save() {
```

Comment:
It's strange that the `Load` and `Delete` methods for `RunState` are functions, but saving is a method. 
Can we just leave everything as functions here?

```go
package configuration

func saveRunstate(r *Runstate) {...}
```

then call via

```go
configuration.saveRunstate(runState)